sycl/test/lit.cfg.py

```diff
@@ -68,7 +68,7 @@
     config.environment['OCL_ICD_FILENAMES'] = os.environ['OCL_ICD_FILENAMES']
 
 config.substitutions.append( ('%clang_cc1', ' ' + config.clang + ' -cc1 ') )
-config.substitutions.append( ('%clangxx', ' ' + config.clangxx + ' -I'+config.opencl_include ) )
+config.substitutions.append( ('%clangxx', ' ' + config.clangxx + ' -I'+config.opencl_include + ' -I'+config.sycl_source_dir ) )
```

Comment:
I think all clang variables must have the same set of include search paths.
Please, apply to `%clang_cl` and `%clang`.
I also think we should not use `%clang_cc1` in SYCL runtime library tests, so it's better to remove this variable from sycl lit config file.

Reply:
@bader I'm going to rewrite tests that use something from source dir as unit tests. Then I'll remove this flag. Personally, I'm not ok with end-to-end tests accessing some internals. Thus, I see no point in adding aforementioned include path to other commands. `%clang_cc1` will also be removed in a separate patch. Are you ok with that?

Reply:
Are you going to rewrite the tests in the scope of this PR or separately?

Reply:
Separately